I'm sorry, but an uncaught exception occurred.

Compiling ATL code at game/image_defs.rpy:9
  File "game/script.rpy", line 128, in script
    show beach_draft
Exception: Not a displayable: <store.TintMatrix object at 0x0000000008cc1d60>

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/script.rpy", line 128, in script
    show beach_draft
  File "renpy/ast.py", line 1357, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/ast.py", line 730, in show_imspec
    renpy.config.show(
    ~~~~~~~~~~~~~~~~~^
        name, at_list=at_list, layer=layer, what=expression, zorder=zorder, tag=tag, behind=behind, atl=atl
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "renpy/exports/displayexports.py", line 513, in show
    sls.add(
    ~~~~~~~^
        layer,
        ^^^^^^
    ...<8 lines>...
        transient=transient,
        ^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "renpy/display/scenelists.py", line 464, in add
    thing.visit_all(lambda d: None)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
[Previous line repeated 1 more time]
  File "renpy/display/displayable.py", line 423, in visit_all
    for d in self.visit():
             ~~~~~~~~~~^^ 
  File "renpy/display/layout.py", line 1602, in visit
    self.update(self.last_st, self.last_at)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/display/layout.py", line 1620, in update
    child = raw_child._duplicate(self._args)
            ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "renpy/display/transform.py", line 1215, in _duplicate
    rv.take_execution_state(self)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "renpy/atl.py", line 554, in take_execution_state
    block = self.compile()
            ~~~~~~~~~~~~^^
  File "renpy/atl.py", line 732, in compile
    block = self.atl.compile(self.context)
            ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "renpy/atl.py", line 926, in compile
    statements = [i.compile(ctx) for i in self.statements]
                  ~~~~~~~~~^^^^^                          
  File "renpy/atl.py", line 1234, in compile
    child = renpy.easy.displayable(child)
            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "renpy/easy.py", line 142, in displayable
    raise Exception("Not a displayable: %r" % (d,))
Exception: Not a displayable: <store.TintMatrix object at 0x0000000008cc1d60>

Windows-11-10.0.26200-SP0 AMD64
Ren'Py 8.4.1.25072401
The Lost Isle 1.0
Fri Oct 24 14:17:04 2025
